Core/Entities/Post.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Core.Entities
+{
+    public class Post : EntityBase
+    {
+        public string Message { get; set; }
+        public string Preview { get; set; }
+
+        [MaxLength(132)]
```

Comment:
Personal preference - you should leave data storage configuration out of core entity classes. Use Fluent API in the `Infrastructure` project instead.